### PR TITLE
Add "IATA Code Decoder" extension

### PR DESCRIPTION
**Pull request by timrogers**
## Description

This PR introduces a new Raycast extension, IATA Code Decoder, for decoding [IATA][1] codes for airports, airlines and aircraft.

The extension contains a command for airports, a command for airlines and a command for aircraft.

When you enter an IATA code, it'll tell you what that code refers to (for example, the airport code `SFO` refers to San Francisco International Airport). It includes typeahead functionality to give helpful suggestions - for example, when you enter `77` as an aircraft code, it'll show you all aircraft with codes beginning in `77` - the Boeing 77- family.

The data is sourced from a [bespoke API][2].

[1]: https://iata.org
[2]: https://github.com/timrogers/iata-code-decoder-api

## Screencast

See the screencast on [Loom](https://www.loom.com/share/656d997b666849329f7ad34adf35bf8b)

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed in the `metadata` folder

### CODE REVIEW

File: extensions/iata-code-decoder/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# IATA Code Decoder Changelog
+
+## Hello 👋 - (2022-03-20)
```
Comment (pernielsentikaer) [inline extensions/iata-code-decoder/CHANGELOG.md:3]:
```suggestion
## [Hello 👋] - 2022-03-20
```
We changed the format of the [changelog](https://developers.raycast.com/basics/prepare-an-extension-for-store#version-history)

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @timrogers 👋 

Thanks for your contribution 🔥

I only left a single suggestion for you to address, looking forward to testing it out again 💪 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by timrogers** [comment]
> Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

Thanks ❤️ Good to go!

**Comment by pernielsentikaer** [comment]
Hi @timrogers 👋 

How often do you expect the data to change?

I can see you have some GitHub actions taking care of the data as it is right now (but they seem to fail with error 401).

If data don't change very frequently, could it then be an idea to just make a new PR with the data instead?

**Comment by timrogers** [comment]
👋 They don't change very frequently, but I'd rather not be responsible for keeping the Raycast repo up to date when they do. 

Can we stick with the API, and I'll fix the GitHub Actions? I just haven't configured the secret.

**Pull request merged**